src/App.js

```diff
@@ -1,9 +1,21 @@
+import Mockman from "mockman-js";
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import Cart from "./Pages/Cart/Cart";
+import Home from "./Pages/Home/Home";
+import ProductList from "./Pages/ProductList/ProductList";
+import Wishlist from "./Pages/Wishlist/Wishlist";
```

Comment:
Have a index.js file in pages folder and import all your pages there and from there export all the pages, so this will keep your imports clean eg:
import {Cart, Home, ProductList, Wishlist} from "./Pages"